src/Microsoft.PowerShell.ConsoleHost/host/msh/ConsoleShell.cs

```diff
@@ -14,6 +14,24 @@ namespace Microsoft.PowerShell
     public static
     class ConsoleShell
     {
+        /// <summary>Entry point in to ConsoleShell. This method is called by main of minishell.</summary>
```

Comment:
Is the second sentence here still correct, or should we update this to indicate its primary intended use case?